docs/actions/ssh.md

```diff
@@ -0,0 +1,264 @@
+!!! note "Target Authentication"
+
+    You can either use basic authentication or use a private key and a passphrase. 
+
+    ```json title="target example"
+    {
+        "name": "my_target",
+        "url": "tcp://my.target:4242",
+        "properties": {
+            "username": "myUsername", // (1)
+            "password": "myPassword", // (2)
+            "privateKey": "/path/to/the/private/key", // (3)
+            "privateKeyPassphrase": "myPrivateKeyPassphrase" // (4)
+        }
+    }
+    ```
+
+    1. Valid properties are `username` or `user`. Set this for basic authentication
+    2. Valid properties are `userPassword` or `password`. Set this for basic authentication
+    3. Path to private key file on the machine running Chutney
+    4. Valid properties are `privateKeyPassword` or `privateKeyPassphrase`. Set this for basic authentication
+
+# SCP
+
+## Download
+
+=== "Inputs"
+
+    | Required | Name          | Type                                               | Default |
+    |:--------:|:--------------|:---------------------------------------------------|:-------:|
+    |    *     | `target`      | String                                             |         |
+    |    *     | `source`      | String                                             |         |
+    |    *     | `destination` | String                                             |         |
+    |          | `timeout`     | String ([Duration](/actions/other/#duration-type)) | `5 sec` |
+
+SCP download has no outputs
```

Comment:
SCP download has no output ? (without **s**)

Reply:
Yes : https://github.com/chutney-testing/chutney/blob/bde8614ca9ca84e75a69ecb9253e8768b407c1f9/task-impl/src/main/java/com/chutneytesting/task/ssh/SftpDownloadTask.java#L50

It uses the source and destination path, but it does not set the file content into the execution context